src/lib/import_export/binary/binary_parser.cpp

```diff
@@ -285,4 +299,19 @@ std::unique_ptr<const BaseCompressedVector> BinaryParser::_import_offset_value_v
   }
 }
 
+std::shared_ptr<FixedStringVector> BinaryParser::_import_fixed_string_vector(std::ifstream& file, const size_t count) {
+  const auto string_length = _read_value<uint32_t>(file);
+  std::vector<pmr_string> values(count);
+
+  for (uint32_t row = 0; row < count; row++) {
+    std::ostringstream value;
+    for (uint32_t c = 0; c < string_length; c++) {
+      value << _read_value<char>(file);
+    }
+    values[row] = pmr_string(value.str());
+  }
+
+  return std::make_shared<FixedStringVector>(values.begin(), values.end(), string_length);
+}
+
```

Comment:
This looks quite expensive. Feel free to add a new (private and friended?) constructor to FixedStringVector. Something like this:

```
const auto string_length = _read_value<uint32_t>(file);

pmr_vector<char> v(string_length * count);
file.read(&v[0], string_length * count);
return std::make_shared<FixedStringVector>(std::move(v), string_length);
```

Reply:
When making this constructor non-public, we cannot call `make_shared`, right?

Reply:
Mmh, ok. Making it public probably doesn't hurt.